### ISSUE TITLE
Do u have any plan to support different locale alphabets

### ISSUE THREAD

**Issue by Moozart** [darkokoa/compose-datetime-wheel-picker#5]
Very useful library. Thank you. 
Do u have any plan to support different locale alphabets?

**Comment by darkokoa** [comment]
Yes, I plan to support it. I will implement it in the near future.

**Comment by Moozart** [comment]
Thanks to you for this libarary.
Do u have any estimation date about it? I am using this library now but if it will takes a long time to implement locale feature, I should migrate to other calendars which is support locale as a temporary
